common/CONSTANTS.ts

```diff
@@ -20,7 +20,7 @@ export default {
 	collectorTime: 45_000,
 	zeroWidthSpace: "\u200b",
 	emojis: {
-		statuses: { yes: "<:yes:1016127835217334322>", no: "<:no:1016127863273037935>" },
+		statuses: { yes: "<:yes:1040475228226203659>", no: "<:no:1040475229920706620>", on: "<:On:1040860528203018310>", off: "<:Off:1040860529071239230>" },
```

Comment:
Please don't change the Yes and No emojis. Please upload the on and off emojis to the Scradd server.

Reply:
oh crap I forgot to not change those